docs/source/usersguide/depletion.rst

```diff
@@ -177,3 +192,163 @@ across all material instances.
+                                                          nuclides,
+                                                          micro_xs,
+                                                          chain_file,
+                                                          nuc_units='atom/cm3')
+
+A user can then define an integrator class as they would for a coupled
+transport-depletion calculation and follow the same steps from there.
+
+.. note::
+
+   Ideally, one-group cross section data should be available for every
+   reaction in the depletion chain. If a nuclide that has a reaction 
+   associated with it in the depletion chain is present in the `nuclides` 
+   parameter but not the cross section data, that reaction will not be
+   simulated.
+
+Generating Microscopic Cross Sections
+-------------------------------------
+
+Users can generate the one-group microscopic cross sections needed by
+:class:`~openmc.deplete.IndependentOperator` using the
+:class:`~openmc.deplete.MicroXS` class::
+
+    import openmc
+
+    model = openmc.Model.from_xml()
+
+    micro_xs = openmc.deplete.MicroXS.from_model(model, model.materials[0])
+
+The :meth:`~openmc.deplete.MicroXS.from_model()` method will produce a
+:class:`~openmc.deplete.MicroXS` object with microscopic cross section data in
+units of ``b``, which is what :class:`~openmc.deplete.IndependentOperator`
+expects the units to be. The :class:`~openmc.deplete.MicroXS` class also includes functions to read in cross section data directly from a ``.csv`` file or from data arrays::
+
+    micro_xs = MicroXS.from_csv(micro_xs_path)
+
+    nuclides = ['U234', 'U235', 'U238']
+    reactions = ['fission', '(n,gamma)']
+    data = np.array([[0.1, 0.2],
+                     [0.3, 0.4],
+                     [0.01, 0.5]])
+    micro_xs = MicroXS.from_array(nuclides, reactions, data)
+
+.. important ::
+
+   Both :meth:`~openmc.deplete.MicroXS.from_csv()` and
+   :meth:`~openmc.deplete.MicroXS.from_array()` assume the cross section values 
+   provided are in barns by defualt, but have no way of verifying this. Make
+   sure your cross sections are in the correct units before passing to a
+   :class:`~openmc.deplete.IndependentOperator` object.
+
+Caveats
+-------
+
+Reaction Rate Normalization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~openmc.deplete.IndependentOperator` class supports two methods for
+normalizing reaction rates:
+
+.. important::
+
+   Make sure you set the correct parameter in the :class:`openmc.abc.Integrator`
+   class. Use the ``source_rates`` parameter when
+   ``normalization_mode == source-rate``, and use ``power`` or ``power_density``
+   when ``normalization_mode == fission-q``.
+
+1. ``soure-rate`` normalization, which assumes the ``source-rate`` provided by
+   the time integrator is a flux, and obtains the reaction rates by multiplying
+   the cross-sections by the ``source-rate``.
+2. ``fission-q`` normalization, which assumes the ``source-rate`` provided by
```

Comment:
In this case, doesn't the user pass `power` or `power_density`, not `source-rate`?

Reply:
Correct. I think I was referring to how internally in `Integrator`, the variable passed to the operator classes no matter what the user provides is `source_rate`, but that doesn't really make a difference to the user. I'll adjust the wording here.